src/client/styles.css

```diff
@@ -819,133 +821,81 @@ body {
 
 /* Smaller title "Find Your" */
 .landing-container .title-find-your {
-    font-size: 2rem;
-    margin-bottom: 0;
+    font-size: 2.4rem;
+    margin-bottom: -2rem;
 }
 
 /* Larger title "Next Home" */
 .landing-container .title-next-home {
     font-size: 4rem;
-    margin-top: 0;
 }
 
 /* Style for the Join button */
 .landing-container .join-button {
     align-self: flex-start; /* Align the button to the left */
     margin-top: 1rem;
-    font-size: 1.5rem;
-    padding: 1rem 2rem;
-    background-color: #FFD700; /* Yellow color */
+    font-size: 1.2rem;
+    padding: .7rem 0;
+    background-color: #F4D06F; /* Yellow color */
     border: none;
     border-radius: 2rem;
     cursor: pointer;
 }
 
-/* Style for SignInView */
-
-/* Styles for all input fields within .text-input in SignInView */
-.sign-in-container .text-input input {
-    border: 2px solid #ccc; /* Make border thicker */
-}
+/* CreateAccountContainer.js */
 
-/* Highlight all focused input fields within .text-input in SignInView */
-.sign-in-container .text-input input:focus {
```

Comment:
The TextInput component already has this styling feature so I wasn't sure if this was intentional.